sharness/t0030-ctl-pin.sh

```diff
@@ -49,6 +49,22 @@ test_expect_success IPFS,CLUSTER "wait for data to unpin from cluster with ctl w
     ipfs-cluster-ctl status "$cid" | grep -q -i "UNPINNED"
 '
 
+test_expect_success IPFS,CLUSTER "pin and unpin data to cluster with ctl using ipfs/ipns/ipld paths" '
```

Comment:
If you want to separate pin and unpin(the way it has been done in previous tests here), this would be a bit  tricky. The test for pinning is fine, but while removing it we only know hash of the file (using ipfs ls), but not the path.  I think it would make more sense to separate tests for ipfs path and ipns path

Reply:
not that tricky, I can just use `ipfs add -n` to get hash